lib/auth/init.go

```diff
@@ -320,7 +333,7 @@ func Init(ctx context.Context, cfg InitConfig, opts ...ServerOption) (*Server, e
 	// singletons). However, we need to keep them around while Telekube uses them.
```

Comment:
What is Telekube?

Reply:
I _think_ that was the name of teleport when it was running as a component of Gravity.  Some deep cuts.